packages/apps/crm/src/components/organisms/TourLineActionCard/TourLineActionCard.tsx

```diff
@@ -49,64 +49,54 @@ const TourLineActionCard = ({
   version,
   selectedStatus,
 }: TourLineActionCardProps) => {
+  const I18n = useTranslator();
   const Colors = useThemeColor();
   const dispatch = useDispatch();
   const navigation = useNavigation();
 
   const [addPopupIsVisible, setAddPopupIsVisible] = useState(false);
 
   return (
-    <View style={styles.globalContainer}>
-      <TourLineCard
-        style={styles.objectCardContainer}
-        name={partner?.fullName}
-        address={address}
-        isValidated={isValidated}
-      />
-      <View style={styles.flexOne}>
-        <CardIconButton
-          style={styles.flexOne}
-          iconName="geo-alt-fill"
-          iconColor={Colors.primaryColor.foreground}
-          onPress={() => linkingProvider.openMapApp(address)}
-        />
-        {eventId != null ? (
-          <CardIconButton
-            style={styles.flexOne}
-            iconName="calendar-event"
-            iconColor={Colors.primaryColor.foreground}
-            onPress={() =>
+    <>
+      <ActionCard
```

Comment:
There is a small display issue when actions are open : 
![image](https://github.com/user-attachments/assets/9c9c60f1-f55c-4fd0-9138-f5830a0b3771)